src/VbaCompiler/VbaCompiler.cs

```diff
@@ -52,7 +52,16 @@ public void AddThisDocument(string path)
             this.modules.Add(document);
         }
 
-        public string CompileVbaProject(string intermediatePath, string projectFilename)
+        public MemoryStream CompileVbaProject()
+        {
+            var vbaProjectMemory = new MemoryStream();
+            this.CompileVbaProject(vbaProjectMemory);
+            vbaProjectMemory.Position = 0;
```

Comment:
In [.NET documentation for MemoryStream class](https://learn.microsoft.com/en-us/dotnet/api/system.io.memorystream?view=net-7.0), the recommended way to set the position of the stream is to use the `Seek` method

`memStream.Seek(0, SeekOrigin.Begin);` 

But this code is definitely more readable.

Reply:
@m-kovac basically, `Seek` does the same thing (with some additional checks): https://github.com/microsoft/referencesource/blob/51cf7850defa8a17d815b4700b67116e3fa283c2/mscorlib/system/io/memorystream.cs#L482